giraffez/commandline.py

```diff
@@ -322,8 +322,8 @@ def run(self, args):
                 if f.file_type == GIRAFFE_ARCHIVE_FILE:
                     encoder = _encoder.Encoder(columns)
                     processors.append(encoder.unpack_row)
-                else:
-                    processors.append(text_to_strings(src_delimiter))
+                else: # text_to_string is handled by the csv parser
```

Comment:
Left it only because it may be a place to plug in further file type splitting, e.g. JSON.  But of course it can be added back in if and when.